storm-server/src/main/java/org/apache/storm/daemon/supervisor/Supervisor.java

```diff
@@ -135,7 +135,7 @@ public Supervisor(Map<String, Object> conf, IContext sharedContext, ISupervisor
             (String) conf.get(DaemonConfig.SUPERVISOR_AUTHORIZER), conf);
         if (authorizationHandler == null && conf.get(DaemonConfig.NIMBUS_AUTHORIZER) != null) {
             throw new IllegalStateException("It looks like authorization is turned on for nimbus but not for the "
-                                            + "supervisor....");
+                + "supervisor.... ( " + DaemonConfig.SUPERVISOR_AUTHORIZER + " is not set)");
```

Comment:
Nit: The `....` part seems a little awkward. Maybe replace with a single period.

Reply:
Will do on checkin.